cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceConfigMapUtilsTest.java

```diff
@@ -0,0 +1,173 @@
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.COMPLETED_BYTE_MOVEMENT_PERCENTAGE_KEY;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.ESTIMATED_TIME_TO_COMPLETION_IN_MINUTES_KEY;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.EXECUTOR_STATE_KEY;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.TIME_COMPLETED;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceConfigMapUtils.updateRebalanceConfigMapWithProgressFields;
+import static io.strimzi.operator.cluster.operator.assembly.KafkaRebalanceStatusTest.buildOptimizationProposal;
+import static io.strimzi.operator.cluster.operator.resource.cruisecontrol.ExecutorStatusTest.createExecutorStatusJson;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class KafkaRebalanceConfigMapUtilsTest {
+
+    private static final String BROKER_LOAD = buildOptimizationProposal().toString();
+    private static final Map<String, String> BROKER_LOAD_MAP = Map.of(BROKER_LOAD_KEY, buildOptimizationProposal().toString());
+
+    private static ConfigMap createKafkaRebalanceConfigMap(Map<String, String> brokerLoad) {
+        return new ConfigMapBuilder()
+                .addToData(brokerLoad)
+                .build();
+    }
+
+    @Test
+    public void testProgressFieldsForProposalReadyState() throws Exception {
+        ExecutorStatus es = new ExecutorStatus(createExecutorStatusJson(
+                CruiseControlExecutorState.NO_TASK_IN_PROGRESS.toString(), null, null, null));
+        ConfigMap cm = createKafkaRebalanceConfigMap(BROKER_LOAD_MAP);
+        updateRebalanceConfigMapWithProgressFields(KafkaRebalanceState.ProposalReady, es, cm);
+
+        Map<String, String> m =  cm.getData();
+        assertThat(m.containsKey(ESTIMATED_TIME_TO_COMPLETION_IN_MINUTES_KEY), is(false));
+        assertThat(m.get(COMPLETED_BYTE_MOVEMENT_PERCENTAGE_KEY), is("0"));
+        assertThat(m.containsKey(EXECUTOR_STATE_KEY), is(false));
+        assertThat(m.get(BROKER_LOAD_KEY), is(BROKER_LOAD));
+    }
+
+    @Test
+    public void testProgressFieldsForProposalRebalancingState() throws Exception {
+        Instant taskStartTime = Instant.now().minusSeconds(2).truncatedTo(ChronoUnit.SECONDS);
+        ExecutorStatus es0 = new ExecutorStatus(createExecutorStatusJson(
+                CruiseControlExecutorState.INTER_BROKER_REPLICA_MOVEMENT_TASK_IN_PROGRESS.toString(), "250", "10000",
+                String.format("No reason provided (Client: 172.17.0.1, Date: %s)", taskStartTime.toString())));
+        ConfigMap cm = createKafkaRebalanceConfigMap(BROKER_LOAD_MAP);
+        updateRebalanceConfigMapWithProgressFields(KafkaRebalanceState.Rebalancing, es0, cm);
+
+        /**
```

Comment:
You shouldn't have javadoc comments inside methods

Reply:
Fixed into block comments.